pytorch/pytorch
# ns for fx: more meaningful error message when creating shadow model

Stack from [ghstack](https://github.com/ezyang/ghstack):
* #76663 ns for fx: skip shadowing ops if copy subgraph is not implemented
* #76561 ns for fx: skip shadowing for torch.cat, and also for nodes with only kwargs
* #76504 ns for fx: skip operator.add and operator.mul when shadowing
* **#76468 ns for fx: more meaningful error message when creating shadow model**

Summary:

This makes the error message when copying an unsupported node more verbose.
This is useful to debug where specifically in a user model this is failing.

Test plan:

1. hardcode this condition to hit
2. run NS tests
3. verify the exception now prints details about the offending node

Differential Revision: [D35978652](https://our.internmc.facebook.com/intern/diff/D35978652)
